root/components/common-macros.tt

```diff
@@ -191,7 +191,7 @@ END -%]
     # parameters: entity, text, no_escape
       SET text = html_escape(text) UNLESS no_escape;
       SET text = html_escape(entity.name) IF text == '';
-      SET text = html_escape(l('[removed]')) IF text == '';
+      SET text = html_escape(lp('[removed]', 'generic, singular')) IF text == '';
```

Comment:
1. It doesn’t seem needed to specify “singular” when there is no ambiguity.
2. To follow the now merged PR https://github.com/metabrainz/musicbrainz-server/pull/3161, it might be worth using the same “generic entity” context key.

```suggestion
      SET text = html_escape(lp('[removed]', 'generic entity')) IF text == '';
```